### ISSUE TITLE
No feedback on successful change of rubric settings

### ISSUE THREAD

**Issue by jonatanschroeder** [PrairieLearn/PrairieLearn#14051]
### Details

When changing the points of a rubric item in the settings card, when I click on Save, there is no feedback to indicate that the settings were saved. The grading panel is updated, but if the changes are small and there are many rubric items, this is not easily seen. There should be some feedback that save worked (popover, toast, disable save button, etc.).
